mitiq/conversions.py

```diff
@@ -96,10 +104,10 @@ def convert_from_mitiq(circuit: Circuit, conversion_type: str) -> QPROGRAM:
         from mitiq.mitiq_pyquil.conversions import to_pyquil
 
         conversion_function = to_pyquil
-    elif isinstance(circuit, Circuit):
+    elif conversion_type == "cirq":
 
         def conversion_function(circ: Circuit) -> Circuit:
-            return circ
+            return Circuit(circ)
```

Comment:
```suggestion
            return circ
```
Here we could return just `circ` if we unfreeze before conversions. Alternatively `Circuit(circ, device=circ.device)` is perhaps safer than `Circuit(circ)`.